ap/columnar_util.py

```diff
@@ -36,9 +36,12 @@
 pq = maybe_import("pyarrow.parquet")
 
 
-#: columns that are always required when opening a nano file with coffea
+#: Columns that are always required when opening a nano file with coffea.
 mandatory_coffea_columns = {"run", "luminosityBlock", "event"}
 
+#: Empty-value definition in places where a number is expected.
+EMPTY = -99999
```

Comment:
Just to make sure: This quantity is then also accessible to and can be loaded by other modules, right? It might be nice to have this globally accessible :+1: